### ISSUE TITLE
Does it support webrtc or how can i enable webrtc for it?

### ISSUE THREAD

**Issue by softopya** [pwnall/chromeview#11]
I need webrtc support. Can we do that?

**Comment by twoh** [comment]
Yes I am too... also need this for webrtc. Does the newest release support webrtc?

**Comment by pwnall** [comment]
I haven't been able to work on this code for a while. However, I'm reasonably sure that ChromeView uses the same underlying code (android_webview) as the Chrome-powered WebView in Android 4.4. Since WebView doesn't support WebRTC, I'm guessing it won't work in ChromeView either.

**Comment by twoh** [comment]
I see... but native Chrome for Android app support WebRTC. Hmm...
